icons/pfsense/pfsense-original.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M10.734.004C4.79.004 0 4.949 0 10.894V105.7l17.34-61.601h17.508l-2.309 8.093h.328c.988-1.156 2.309-2.148 3.961-3.3 1.649-1.16 3.305-2.153 5.129-3.141 1.813-.992 3.793-1.648 5.941-2.309 2.149-.664 4.297-.832 6.446-.832 4.62 0 8.422.828 11.718 2.313 3.305 1.488 5.778 3.8 7.762 6.605 1.649 2.48 2.645 5.453 3.137 8.918l.027-.011c.032.32.137.718.137 1.007.332 3.801-.164 8.09-1.484 12.715-1.16 4.293-2.965 8.426-5.446 12.39a52.642 52.642 0 0 1-8.422 10.4 43.382 43.382 0 0 1-10.906 7.265c-3.965 1.82-8.09 2.648-12.55 2.648a30.202 30.202 0 0 1-10.57-1.82c-3.138-1.156-5.29-3.3-6.606-6.277h-.332l-8.258 29.234h104.543a10.883 10.883 0 0 0 7.715-3.187 10.87 10.87 0 0 0 3.191-7.711V25.105c-.5-.164-.992-.328-1.324-.496a14.91 14.91 0 0 0-2.973-.336c-2.973 0-5.453.665-7.43 1.985-1.82 1.32-3.472 3.965-4.628 8.094l-2.805 9.738h10.734l2.64 8.426-6.937 6.773H104.54L91.5 105.535H72.672l13.05-46.246H78.45l-.246.164h-.246v-.164l4.29-15.2h7.593l2.976-10.898c.989-3.296 2.145-6.597 3.633-9.574 1.32-2.972 3.301-5.617 5.614-7.93 2.312-2.312 5.285-4.128 8.753-5.449 3.47-1.32 7.934-1.98 13.051-1.98 1.324 0 2.473-.004 3.797.164A11.1 11.1 0 0 0 117.258.004Zm35.004 59.285c-4.785 0-8.91 1.488-12.379 4.297-3.468 2.809-5.78 6.605-7.101 11.227-1.32 4.624-1.153 8.421.832 11.226 1.648 2.973 4.953 4.3 9.902 4.3 4.793 0 8.918-1.491 12.387-4.3 3.469-2.805 5.941-6.602 7.266-11.227 1.32-4.62.988-8.417-.829-11.226-1.816-2.809-5.28-4.297-10.07-4.297Zm0 0"/></svg>
```

Comment:
Add ` fill="#1a1a1a"` and newline at the end of the file (needed for some editors)
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#1a1a1a" d="M10.734.004C4.79.004 0 4.949 0 10.894V105.7l17.34-61.601h17.508l-2.309 8.093h.328c.988-1.156 2.309-2.148 3.961-3.3 1.649-1.16 3.305-2.153 5.129-3.141 1.813-.992 3.793-1.648 5.941-2.309 2.149-.664 4.297-.832 6.446-.832 4.62 0 8.422.828 11.718 2.313 3.305 1.488 5.778 3.8 7.762 6.605 1.649 2.48 2.645 5.453 3.137 8.918l.027-.011c.032.32.137.718.137 1.007.332 3.801-.164 8.09-1.484 12.715-1.16 4.293-2.965 8.426-5.446 12.39a52.642 52.642 0 0 1-8.422 10.4 43.382 43.382 0 0 1-10.906 7.265c-3.965 1.82-8.09 2.648-12.55 2.648a30.202 30.202 0 0 1-10.57-1.82c-3.138-1.156-5.29-3.3-6.606-6.277h-.332l-8.258 29.234h104.543a10.883 10.883 0 0 0 7.715-3.187 10.87 10.87 0 0 0 3.191-7.711V25.105c-.5-.164-.992-.328-1.324-.496a14.91 14.91 0 0 0-2.973-.336c-2.973 0-5.453.665-7.43 1.985-1.82 1.32-3.472 3.965-4.628 8.094l-2.805 9.738h10.734l2.64 8.426-6.937 6.773H104.54L91.5 105.535H72.672l13.05-46.246H78.45l-.246.164h-.246v-.164l4.29-15.2h7.593l2.976-10.898c.989-3.296 2.145-6.597 3.633-9.574 1.32-2.972 3.301-5.617 5.614-7.93 2.312-2.312 5.285-4.128 8.753-5.449 3.47-1.32 7.934-1.98 13.051-1.98 1.324 0 2.473-.004 3.797.164A11.1 11.1 0 0 0 117.258.004Zm35.004 59.285c-4.785 0-8.91 1.488-12.379 4.297-3.468 2.809-5.78 6.605-7.101 11.227-1.32 4.624-1.153 8.421.832 11.226 1.648 2.973 4.953 4.3 9.902 4.3 4.793 0 8.918-1.491 12.387-4.3 3.469-2.805 5.941-6.602 7.266-11.227 1.32-4.62.988-8.417-.829-11.226-1.816-2.809-5.28-4.297-10.07-4.297Zm0 0"/></svg>

```

Reply:
Seems like the original icon actually has no color, so this change is not needed

Reply:
@yeyeto2788 Actually, we found the official `original` logo, so you can use that instead of this :)
https://github.com/pfsense/pfsense/blob/a7086b04cae21ca742fdeefd1019ee1401b6dded/src/usr/local/www/safari-pinned-tab.svg
![](https://raw.githubusercontent.com/pfsense/pfsense/a7086b04cae21ca742fdeefd1019ee1401b6dded/src/usr/local/www/safari-pinned-tab.svg)